server/moodle/mod/livequiz/classes/models/questions_answers_relation.php

```diff
@@ -37,7 +37,7 @@ class questions_answers_relation {
      * @throws dml_exception
      * @throws dml_transaction_exception
      */
-    public static function append_answer_to_question(int $questionid, int $answerid): void {
+    public static function insert_question_answer_relation(int $questionid, int $answerid): void {
```

Comment:
@adomasciplys We should probably correct our `append_student_to_quiz` to correspond to the new naming scheme.